### PR TITLE
Ashley/bugfixes

### PR DESCRIPTION
**Pull request by ashleynguyen27**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?
various bug fixes listed below:
* changing to correct header colors [all dashboard pages (might be something I missed)]
* announcements scrollable [general dashboard]
* adding the loading component when sending an invite [admin]
* fixing card styling [subscriber dashboard]
* input border [onboarding]
* card container overflow [admin]
* phone number validation [onboarding]
* state validation and error message [onboarding]
* remove outline none [onboarding]

### How to Review
heh 

### Design Status
HIFI

CC: @dfangshuo

### CODE REVIEW

File: src/screens/onboarding/steps/ContactInfoStep.js [hunk 1]
```diff
@@ -139,8 +139,10 @@ class ContactInfoStep extends React.PureComponent {
                   returnHome={returnToHomepage}
                   handleCloseModal={this.handleCloseModal}
                 />
-              ) : (
+              ) : errors.permanentState === '' ? (
```
Comment (cindyzhang12) [inline src/screens/onboarding/steps/ContactInfoStep.js:142]: Is this what you mean by "state validation and error message"? out of curiosity what does this line do :'))))

File: src/screens/subscriber/components/DashboardProjectNews.js [hunk 2]
```diff
@@ -16,7 +15,20 @@ export default class DashboardProjectNewsSection extends React.PureComponent {
     } else if (announcements.length === 0) {
       renderedComponent = <NoProjects />;
     } else {
-      renderedComponent = <AnnouncementList announcements={announcements} />;
+      renderedComponent = announcements.map(announcement => (
+        <div key={announcement.title} className="subscriber-news-card">
+          <div className="cardHeading">
```
Comment (cindyzhang12) [inline src/screens/subscriber/components/DashboardProjectNews.js:20]: change this to kebab case! card-heading :)

File: src/screens/subscriber/components/DashboardProjectNews.js [hunk 3]
```diff
@@ -16,7 +15,20 @@ export default class DashboardProjectNewsSection extends React.PureComponent {
...
+            <h3>{announcement.title}</h3>
+            {announcement.attachments
+              ? announcement.attachments.map(attachment => (
+                  <img src={attachment.url} alt={attachment.filename} />
+                ))
+              : null}
+            <p>{announcement.message}</p>
+          </div>
+          <div className="cardDetails" />
```
Comment (cindyzhang12) [inline src/screens/subscriber/components/DashboardProjectNews.js:29]: kebab case! :)

File: src/screens/onboarding/steps/ContactInfoStep.js (hunk 1 above)
Comment (cindyzhang12) [inline src/screens/onboarding/steps/ContactInfoStep.js:142]: Is this what you mean by "state validation and error message"? out of curiosity what does this line do :'))))

File: src/lib/onboardingUtils.js [hunk 4]
```diff
@@ -98,10 +96,16 @@ const validateZipcode = value => {
   return value.length === 5 ? '' : 'Must be 5 digits';
 };
 
+const validatePhoneNumber = value => {
+  const re = /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im;
```
Comment (cindyzhang12) [inline src/lib/onboardingUtils.js:100]: maybe leave a comment about what this regex is supposed to do for other people who end up looking at this code!

**Comment by dfangshuo** [comment]
please make sure to address linter errors as well

File: src/screens/onboarding/steps/ContactInfoStep.js (hunk 1 above)
Comment (ashleynguyen27) [reply src/screens/onboarding/steps/ContactInfoStep.js:143]: hi yes. this line is just checking whether errors.permanentState is empty. empty = valid input; not empty = error message. This is what shows error messages

Comment (cindyzhang12) [reply src/screens/onboarding/steps/ContactInfoStep.js:143]: ah gotcha!!

**Pull request merged**